casper/src/test/resources/RevVaultTest.rho

```diff
@@ -43,8 +42,7 @@ match (
           [
             ("Create genesis vault with expected balance", *testFindOrCreateGenesisVault),
             ("Return an error for invalid public key", *testCreateVaultFail),
-            ("Create Alice vault and see it is locked", *testCreateAliceVaultLocked),
-            ("Create Alice vault, unlock it and check balance", *testCreateAliceVaultUnlocked),
+            ("Create vault and see it is secured", *testCreateVault),
```

Comment:
Missing "if".

Reply:
I meant `Create vault and see [that] it is secured`, which I understand to be a n omission not changing the meaning. I think I'll clarify though :)